cloud/docker.go

```diff
@@ -68,13 +68,15 @@ func (settings *dockerSettings) Validate() error {
 		return errors.New("Port must not be blank")
 	}
 
-	if settings.PortRange != nil {
-		min := settings.PortRange.MinPort
-		max := settings.PortRange.MaxPort
+	if settings.PortRange == nil {
```

Comment:
Go has a concept of a zero value, which for an integer is 0. If both `settings.PortRange.MinPort` and `settings.PortRange.MaxPort` are 0, then this will validate, but I suspect we won't be able to spawn a host. Should the check on line 78 actually be `<=` instead of `<`?

Reply:
done